libr/core/cbin.c

```diff
@@ -455,7 +456,9 @@ static void _print_strings(RCore *r, RList *list, int mode, int va) {
 			pj_kn (pj, "length", string->length);
 			pj_ks (pj, "section", section_name);
 			pj_ks (pj, "type", type_string);
-			pj_ks (pj, "string", string->string);
+			// data itself may be encoded so use pj_ke
+			pj_ke (pj, "string", string->string);
```

Comment:
keep using pj_ks imho theres no benefit in adding a new api for just strings

Reply:
Fixed everything else in the PR, this is the last thing I'm not clear on.

`pj_ks` is used for strings yes but the main point of this PR was that `pj_ks` can produce invalid json strings (e.g. including `\x` and the user will now get a new way to force those strings to be encoded as base64, hex, stripped, etc.

I guess, either:

1. `pj_ke` is new and is used only in the locations we think the source data could be invalid per the JSON spec (e.g. class names, method names, strings, version info)
2. We just keep using `pj_ks` but add some logic at the start of the function to check if there's an encoding configured, and branch to that logic instead of proceeding with the default string builder.

Option 2. would mean that we are encoding every current data point used by pj_ks which may or may not be desirable - thoughts?